examples/react/trimmed/vite.config.ts

```diff
@@ -6,4 +6,14 @@ import react from '@vitejs/plugin-react';
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [tsconfigPaths(), react(), tailwindcss()],
+  server: {
+    proxy: {
+      // Proxy to bypass CORS on fastopendata.org
+      '/data-proxy': {
```

Comment:
Why do we need a proxy for this? why not instead just connect right to your Bucket/Server?